Source/Streamstone/Stream.Operations.cs

```diff
@@ -31,7 +31,7 @@ public Stream Execute()
 
                 try
                 {
-                    table.Execute(insert.Prepare());
+                    table.ExecuteAsync(insert.Prepare()).Wait();
```

Comment:
@khellang not exactly...

I'm actually in favor of remove all sync code and let the developer choose how to schedule it from the async methods...